gluten-flink/runtime/src/main/java/org/apache/gluten/table/runtime/operators/GlutenSingleInputOperator.java

```diff
@@ -75,7 +83,8 @@ public GlutenCalOperator(String plan, String id, String inputType, String output
     public void open() throws Exception {
         super.open();
         outElement = new StreamRecord(null);
-        session = Velox4j.newSession(MemoryManager.create(AllocationListener.NOOP));
+        memoryManager = MemoryManager.create(AllocationListener.NOOP);
+        session = Velox4j.newSession(memoryManager);
 
         inputQueue = new LinkedBlockingQueue<>();
```

Comment:
Could move to this API https://github.com/velox4j/velox4j/blob/40474f2da7e59dfb1b2bf2682310eec2d2b0ab85/src/main/java/io/github/zhztheplayer/velox4j/connector/ExternalStreams.java#L35-L37 while the Velox4J version is updated in the future.

Reply:
Can we update it in a seperated pull request? As first, these code will not be used for nexmark performance, it should not be used, then I am afraid this interface is still not the final interface, we may still change it later.